sites/public/src/components/listing/ListingView.tsx

```diff
@@ -182,8 +182,15 @@ export const ListingView = (props: ListingProps) => {
       !listingPref.multiselectQuestions.hideFromListing
   )
 
-  const getPreferenceData = () => {
-    return listingPreferences.map((listingPref, index) => {
+  const listingPrograms = listing?.listingMultiselectQuestions.filter(
```

Comment:
Could we combine this w the above section, where you just need to pass in preferences / programs?